docs/serverless/ingest/auto-import.mdx

```diff
@@ -0,0 +1,85 @@
+---
+slug: /serverless/security/automatic-import
+title: Automatic import
+description: Use automatic import to quickly ingest third-party data.
+tags: [ 'serverless', 'security', 'how-to' ]
+status: in review
+---
+
+<DocBadge template="technical preview" />
+
+<DocCallOut title="Technical preview" color="warning">
+This feature is in technical preview. It may change in the future, and you should exercise caution when using it in production environments. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of GA features.
+</DocCallOut>
+
+This page explains how to use Automatic Import. Automatic Import helps you quickly parse, ingest, and create [ECS mappings](https://www.elastic.co/elasticsearch/common-schema) for data from sources that don't yet have prebuilt Elastic integrations. This can accelerate your migration to Elastic's SIEM, and help you quickly add new data sources to an existing SIEM deployment. Automatic Import uses a large language model (LLM) with specialized instructions to quickly analyze your source data and create a custom integration. 
+
+While Elastic has 400+ [prebuilt data integrations](((integrations-docs))), Automatic Import helps you extend data coverage to other security-relevant technologies and applications. Elastic integrations (including those created by Automatic Import) normalize data to [Elastic Common Schema (ECS)](https://www.elastic.co/blog/ecs-elastic-common-schema-otel-opentelemetry-faq), which creates uniformity across dashboards, search, alerts, machine learning, and more. 
```

Comment:
I think I've usually heard it referred to as "the Elastic Common Schema," as the [ECS docs](https://www.elastic.co/guide/en/ecs/current/ecs-reference.html) do. And this link should probably point to those docs, rather than a blog post?

Reply:
Yep! What @joepeeples said ⬆️